package.json

```diff
@@ -60,6 +60,7 @@
     "@buttercup/dropbox-client": "^0.1.1",
     "@buttercup/signing": "^0.1.0",
     "gzip-js": "^0.3.2",
+    "hash.js": "1.1.7",
```

Comment:
This could be `^1.1.7`

Reply:
oh I see, I can remove that part no prob.